tools/proto-compiler/src/functions.rs

```diff
@@ -172,7 +172,7 @@ pub fn copy_files(src_dir: &Path, target_dir: &Path) {
             e.file_type().is_file()
                 && e.file_name()
                     .to_str()
-                    .map(|name| name.starts_with("tendermint."))
+                    .map(|name| name.starts_with(&format!("{project}.")))
```

Comment:
Performance nit: it would be better if the prefix string was formatted once, not with every invocation of the closure.